tests/conditional_compilation/conftest.py

```diff
@@ -171,3 +187,57 @@ def save_session_info(pytestconfig, artifacts):
     """Fixture function for saving additional attributes to configuration file."""
     yield
     write_session_info(path=artifacts / SESSION_INFO_FILE, data=pytestconfig.session_info)
+
+
+@pytest.fixture(scope="function")
+def prepare_models(openvino_ref, models, request):
+    for model in models:
+        if "OMZ" in model["path"]:
+            model["path"] = prepare_omz_model(openvino_ref, model, request)
+    models = [model["path"] for model in models]
+    return models
+
+
+def prepare_omz_model(openvino_ref, model, request):
```

Comment:
Please, add description

Reply:
`request` isn't required in the function. Moreover, it limits the func usage from any other place. Let's provide `omz_path` and `omz_cache_dir` as arguments only